### PR TITLE
fix for issue #2020 - composite summary file

### PR DESCRIPTION
**Pull request by ruvceskistefan**
Related to issue #2020 

Proof of work : https://github.com/ruvceskistefan/git-runner-test/actions/runs/2904902484

image

### CODE REVIEW

**Review by thboop — APPROVED** [review]
lgtm

**Comment by dlemfh** [comment]
Thanks so much for this fix!!

Sort of a newbie question.. In order to use this fix,
do I just wait until the merge commit is released?

Or is there something I need to do on the user-end (of github actions hosted runners)?

Ex. do I need to update a version maybe..?
Thanks again.

**Pull request merged**
